test/browsers-target-test.js

```diff
@@ -0,0 +1,182 @@
+'use strict';
+
+const AddonFactory = require('../');
+const expect = require('chai').expect;
+const helpers = require('broccoli-test-helper');
+const co = require('co');
+
+describe(`Do not include the polyfill if the browser targets match`, function() {
+  let output, subject, addon;
+
+  beforeEach(function() {
+    addon = Object.create(AddonFactory);
+    Object.assign(addon, {
+      addons: [],
+      _fetchBuildConfig: {
+        preferNative: true,
+        browsers: ['last 1 chrome versions']
+      },
+      ui: {
+        writeWarnLine() {
+        }
+      }
+    });
+    subject = addon.treeForVendor();
+    output = helpers.createBuilder(subject);
+  });
+
+  afterEach(co.wrap(function* () {
+    yield output.dispose();
+  }));
+
+  it('preferNative is built into vendor file', co.wrap(function* () {
+    yield output.build();
+    let files = output.read();
+    expect(files).to.have.all.keys('ember-fetch.js');
+    expect(files['ember-fetch.js']).to.include(`var preferNative = true`);
```

Comment:
The result is not correct.

This is my debug output before the line `const polyfillNode = debug(concat(new MergeTrees(inputNodes), {`

```
// chrome devtool
> needsFetchPolyfill
< false

> needsAbortControllerPolyfill
< false

> inputFiles
< (2) ["abortcontroller.js", "fetch.js"]
```